packages/calendar-utils/src/calendar-header/calendar-header.tsx

```diff
@@ -82,7 +82,7 @@ const CalendarHeader = (props: TCalendarHeader) => {
             size="medium"
           />
         </Tooltip>
-        <Text.Body as="span" isBold={true}>
+        <Text.Body as="span" isBold={true} fontWeight="bold">
```

Comment:
I would suggest to already remove the `isBold` usage and keep only the `fontWeight` prop. Having both does not make much sense.

Thanks

Reply:
@emmenko I might have misunderstood but I thought I needed to support both?

Reply:
Makes sense thanks I'll update